app/models/comment.php

```diff
@@ -1,19 +1,40 @@
 <?php
 class Comment extends AppModel
 {
+    CONST FIRST_COMMENT   = 1;
+    CONST MIN_BODY_LENGTH = 1;
+    CONST MAX_BODY_LENGTH = 200;
+    CONST TABLE           = 'comment';
```

Comment:
Changed it po to **TABLE_NAME** https://github.com/renzosunico/MyClassroom/commit/812c3d36d3f44e79df5144435b057ed228025feb